src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/Https/HttpsTests.cs

```diff
@@ -166,15 +222,28 @@ public static void SameBinding_Soap12_EchoString()
         Assert.True(errorBuilder.Length == 0, "Test case FAILED with errors: " + errorBuilder.ToString());
     }
 
-    [ConditionalFact(nameof(Root_Certificate_Installed), nameof(Client_Certificate_Installed))]
-#if FEATURE_NETNATIVE
+#if FULLXUNIT_NOTSUPPORTED
+    [Fact]
     [ActiveIssue(959)] // Server certificate validation not supported in NET Native
 #else
+    [ConditionalFact(nameof(Root_Certificate_Installed), nameof(Client_Certificate_Installed))]
     [ActiveIssue(959, PlatformID.AnyUnix)] // Server certificate validation not supported on Linux and OS X
```

Comment:
According to 959, this should have been fixed for *nix already. 959 should be removed in #else.